### PR TITLE
Add : pyspoa

### PR DESCRIPTION
**Pull request by cjw85**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/pyspoa/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  version: "{{ version }}"
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cmake
```
Comment (bgruening) [inline recipes/pyspoa/meta.yaml:15]: Can you move cmake into the build section?

File: recipes/pyspoa/build.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+git clone --recursive https://github.com/nanoporetech/pyspoa.git
```
Comment (bgruening) [inline recipes/pyspoa/build.sh:3]: Can you maybe create a release tarball with the submodule included? This way we can create a backup of the tarball.

**Review by bgruening — COMMENTED** [review]
recipes/pyspoa/v0.0.3.tar.gz needs to be removed

Thanks @cjw85!

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Sorry, hit the wrong button. @cjw85 if this only fails with py27 feel free to skip python 2.7 if you like.

File: recipes/pyspoa/build.sh (hunk 2 above)
Comment (cjw85) [reply recipes/pyspoa/build.sh]: Done.

**Comment by cjw85** [comment]
@bgruening 

Any idea why the macOS build has errors with setuptools?

```.06:37:15 BIOCONDA INFO (ERR) Traceback (most recent call last):
06:37:15 BIOCONDA INFO (ERR)   File "setup.py", line 2, in <module>
06:37:15 BIOCONDA INFO (ERR)     import setuptools
06:37:15 BIOCONDA INFO (ERR) ModuleNotFoundError: No module named 'setuptools'
06:37:15 BIOCONDA INFO (ERR) make: *** [build] Error 1```

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
@bioconda-bot please merge

**Pull request merged**